### PR TITLE
Link to the debug wiki from README and the command's error message box

### PR DESCRIPTION
**Pull request by testforstephen**
Signed-off-by: Jinbo Wang <jinbwan@microsoft.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,8 +80,8 @@ Where `<your-image-prefix-here>` is something like `docker.io/brendanburns`.
...
    * `Kubernetes: Terminal` - Open an interactive terminal session in a pod of the Kubernetes Deployment
    * `Kubernetes: Exec` - Run a command in a pod of the Kubernetes Deployment
-   * `Kubernetes: Debug (Launch)` - Run the current application as a Kubernetes Deployment and attach a debugging session to it (currently works only for Java/Node.js deployments)
-   * `Kubernetes: Debug (Attach)` - Attach a debugging session to an existing Kubernetes Deployment (currently works only for Java deployments)
+   * `Kubernetes: Debug (Launch)` - Run the current application as a Kubernetes Deployment and attach a debugging session to it (currently works only for Java/Node.js deployments, and refers to the wiki [Debug support on Kubernetes cluster](https://github.com/Azure/vscode-kubernetes-tools/blob/master/debug-on-kubernetes.md) for more details)
+   * `Kubernetes: Debug (Attach)` - Attach a debugging session to an existing Kubernetes Deployment (currently works only for Java deployments, and refers to the wiki [Debug support on Kubernetes cluster](https://github.com/Azure/vscode-kubernetes-tools/blob/master/debug-on-kubernetes.md) for more details)
```
Comment (itowlson) [inline README.md]: Same comment as for Launch command.

File: src/debug/debugSession.ts [hunk 2]
```diff
@@ -49,7 +52,10 @@ export class DebugSession implements IDebugSession {
         // TODO: Support docker-compose.yml
         const dockerfilePath = path.join(workspaceFolder.uri.fsPath, "Dockerfile");
         if (!fs.existsSync(dockerfilePath)) {
-            vscode.window.showErrorMessage(`No Dockerfile found in the workspace ${workspaceFolder.name}`);
+            const answer = await vscode.window.showErrorMessage(`No Dockerfile found in the workspace ${workspaceFolder.name}, please refer to the wiki for the usage.`, "Open in Browser");
```
Comment (itowlson) [inline src/debug/debugSession.ts]: Avoid run-on sentence, and don't refer to the article as a 'wiki'. E.g. `No Dockerfile found in the workspace ${…}. See the documentation for how to use this command.`

File: src/debug/debugSession.ts [hunk 3]
```diff
@@ -15,6 +16,8 @@ import { shell } from "../shell";
 import { DockerfileParser } from "../docker/dockerfileParser";
 import * as dockerUtils from "../docker/dockerUtils";
 
+const debugWikiUrl = "https://github.com/Azure/vscode-kubernetes-tools/blob/master/debug-on-kubernetes.md";
```
Comment (itowlson) [inline src/debug/debugSession.ts]: Would prefer not to use the term 'wiki' for this.  Maybe `debugCommandDocumentationUrl`?

File: src/debug/debugSession.ts [hunk 4]
```diff
@@ -67,7 +73,10 @@ export class DebugSession implements IDebugSession {
         const containerEnv = {};
         const portInfo = await this.debugProvider.resolvePortsFromFile(dockerfile, containerEnv);
         if (!portInfo || !portInfo.debugPort || !portInfo.appPort) {
-            vscode.window.showErrorMessage("Cannot resolve debug/application port from Dockerfile.");
+            const answer = await vscode.window.showErrorMessage("Cannot resolve debug/application port from Dockerfile, please refer to the wiki for the usage.", "Open in Browser");
```
Comment (itowlson) [inline src/debug/debugSession.ts]: Similar comments e.g. `Cannot resolve debug/application port from Dockerfile. See the documentation for how to use this command.`

File: src/debug/debugSession.ts [hunk 5]
```diff
@@ -174,7 +184,10 @@ export class DebugSession implements IDebugSession {
         // Find the debug port to attach.
         const portInfo = await this.debugProvider.resolvePortsFromContainer(this.kubectl, targetPod, targetContainer);
         if (!portInfo || !portInfo.debugPort) {
-            vscode.window.showErrorMessage("Cannot resolve the debug port to attach.");
+            const answer = await vscode.window.showErrorMessage("Cannot resolve the debug port to attach, please refer to the wiki for the usage.", "Open in Browser");
```
Comment (itowlson) [inline src/debug/debugSession.ts]: Two sentences and rephrase second (e.g. as per comments elsewhere)

Comment (itowlson) [inline src/debug/debugSession.ts]: It seems like the `const answer = …; if (open in browser) { … }` is a repeated pattern - consider extracting a helper method.

File: src/debug/debugSession.ts (hunk 2 above)
Comment (testforstephen) [reply src/debug/debugSession.ts]: Thanks, Nice suggestion. Learned it.

**Pull request merged**